docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,374 @@
+repository. Here's how you get them:
+
+1. Install Git: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+1. Clone the Conjur repository
+   
+   In your terminal application, run:
+   ```sh-session
+   $ git clone https://github.com/cyberark/conjur.git
+   ```
+   This will create a folder called `conjur` in the current directory.
+
+### The Good Part
+
+To start out our experience on a high note, let's get the full Conjur+TLS stack
+up and running so we can inspect it.
+
+The tutorial script will install Conjur and NGINX, configure them to work
+together, and connect a Conjur client to the running server. This is a full
+end-to-end working installation to allow you to see how the pieces fit.
+
+```sh-session
+$ # start the Conjur+NGINX tutorial servers
+$ cd conjur/docs/tutorials/integrations/nginx
+$ ./start.sh
+```
+
+Take a look at the logs for the Conjur or NGINX servers:
+
+```sh-session
+$ # in the 'nginx' directory we navigated to previously:
+$ docker-compose logs conjur
+$ docker-compose logs proxy
+```
+
+Run some commands in the Conjur client:
+
+```sh-session
+$ # in the 'nginx' directory we navigated to previously:
+$ docker-compose exec client bash
+# conjur authn whoami
+# conjur list
+```
+
+#### What's happening here?
+
+When you read the Conjur access logs, you'll note that the Conjur server is
+listening on port 80 (*insecure http*) inside its container. However, that port
+is not exposed except on the local Docker network, so requests from the Internet
+and LAN are unable to reach it.
+
+Meanwhile, the NGINX container is exposing its port 443 (*https*) to the outside
+network and proxying the traffic through to Conjur.
+
+## Breaking Down the Tutorial
+
+These files show how the proxy setup works. Note that, while this tutorial uses
+Docker containers and NGINX, there's nothing magic about those technologies. You
+can replicate the same strategy using a different endpoint such as
+[HAProxy][haproxy-tls] and services that run in virtual machines or on bare
+metal. To create a tutorial that can run as conveniently on a laptop as in the
+cloud, we provide this setup.
+
+[haproxy-tls]: https://www.haproxy.com/documentation/aloha/7-0/haproxy/tls/
+
+### docker-compose.yml
+
+This file declares services to be used in the tutorial. Let's break down each declaration:
+
+```yaml
+database:
+  image: postgres:9.3
+```
+
+Conjur requires a Postgres database to store encrypted secrets and other data.
+This service uses the [official Postgres image][postgres-image] from DockerHub.
+
+#### Production tip
+
+Just as we use a proxy in this tutorial to encrypt & authenticate traffic to the
+Conjur server using TLS, you will also want to use TLS for your production
+Postgres database. If you're using [Amazon RDS][rds], it already has TLS support
+built-in. If you're hosting your own database, you'll want to use a technique
+similar to the one described here in order to secure its traffic.
```

Comment:
This should be reworded to be more clear that this tutorial _does not_ instruct on how to secure the DB

Maybe change it to say "you will also want to use TLS for your production Postgres database - but instructions for doing so are beyond the scope of this tutorial."

Reply:
clarified & added link to Postgres recommendations on this topic